server/internal/repository/image.go

```diff
@@ -17,5 +19,17 @@ type (
 func (r *Repository) UploadImage(ctx context.Context, image io.Reader) (uuid.UUID, error) {
 	imageID := uuid.New()
 	// オブジェクトストレージに画像をアップロードする
+	if _, err := r.client.PutObject(
+		ctx,
+		&s3.PutObjectInput{
+			Bucket:      aws.String("leaq"),
```

Comment:
Bucketはデータを保存する場所の名前みたいなものなんですが、今回は`h25s-04`でお願いします！
```suggestion
			Bucket:      aws.String("h25s-04"),
```